src/index.html

```diff
@@ -7,7 +7,268 @@
+            <p class="lure__content">
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical. <br><br>
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.<br><br>
+              <strong>We will never be late!<br>
+              Guaranteed secure transaction - 100%</strong>
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <!-- gallery -->
+      <div class="lure">
+        <div class="gallery">
+          <h1 class="title">
+            Gallery
+          </h1>
+
+          <div class="gl-block">
+            <div class="gl-block__slide gl-block__slide--left">
+              <img
+                src="./images/photos/1.jpg"
+                alt="Marina Palms"
+                class="gl-block__item"
+              >
+              <img
+                src="./images/photos/2.jpg"
+                alt="North Miami Beach"
+                class="gl-block__item"
+              >
+            </div>
+            <p class="gl-block__name"><a href="">Marina Palms</a> / North Miami Beach, FL 33162</p>
+          </div>
+
+          <div class="gl-block">
+            <div class="gl-block__slide gl-block__slide--center">
+              <img
+                src="./images/photos/3.jpg"
+                alt="Edition Residens"
+                class="gl-block__item"
+              >
+              <img
+                src="./images/photos/4.jpg"
+                alt="Miami Beach"
+                class="gl-block__item"
+              >
+            </div>
+            <p class="gl-block__name"><a href="">Edition Residens</a> / Miami Beach, FL 33139</p>
+          </div>
+
+          <div class="gl-block">
+            <div class="gl-block__slide gl-block__slide--right">
+              <img
+                src="./images/photos/5.jpg"
+                alt="Faena House"
+                class="gl-block__item"
+              >
+              <img
+                src="./images/photos/6.jpg"
+                alt="Miami Beach"
+                class="gl-block__item"
+              >
+            </div>
+            <p class="gl-block__name">
+              <a href="">
+                Faena House
+              </a>
+              / Miami Beach, FL 33140
+            </p>
+          </div>
+
+          <button class="button gl-block__button">View all</button>
+        </div>
+      </div>
+
+      <!-- Consultation -->
+
+      <div class="lure">
+        <h1 class="title">
+          Consultation
+        </h1>
+        <div class="form form__margin">
```

Comment:
First of all, you probably don't need a modifier here. Even if you do, it is named incorrectly. It should have only one underscore.

Reply:
Also, name of this block is confusing since the form itself has the class 'form-app' and the rest of the content inside this block (form) is just text.

Reply:
Ну, я це реалызував не як модифікатор, а як елемент, єдиною функцією котрого є відступ. 
Це пов'язано з тим, що задавати марджин блокові -- практика не бажана.
А от у елемент можна _)

Reply:
Однак, з тим що блоки form i form-app необхідно розділити, т.я. вони різні, я згоден.
✓